common/script/codeHelpers.js

```diff
@@ -0,0 +1,29 @@
+export function $w (s) {
+  return s.split(' ');
+}
+
+export function uuid () {
```

Comment:
I don't think we should use this. I think we should outsource our uuid-ing needs to a node module. @paglias Do you have objections to that? I've used https://www.npmjs.com/package/uuid with the v4 function.